testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java

```diff
@@ -5,27 +5,7 @@
 
 package org.wildfly.test.integration.elytron.oidc.client;
 
-import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SYSTEM_PROPERTY;
```

Comment:
Just a small note, it's good to avoid reordering imports (since it's easier to see the changes related specifically to this PR).

Reply:
ok

Reply:
Also, the existing import order is the correct one for WildFly, so please restore it.

Reply:
done